index.js

```diff
@@ -37,6 +39,38 @@ module.exports.filter = shouldCompress
 
 var cacheControlNoTransformRegExp = /(?:^|,)\s*?no-transform\s*?(?:,|$)/
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = false
+try {
+  hasBrotliSupport = 'brotli' in require('process').versions
```

Comment:
The process module is just a global so no need to require it or have in a try catch.